toolchains/jp_workspaces/WORKSPACE.jp60

```diff
@@ -0,0 +1,92 @@
+workspace(name = "Torch-TensorRT")
```

Comment:
Use the MODULE.bazel format now

Reply:
@narendasan 
since there is only torch2.4 nvidia wheel downloadable, so I tested this change on top of the release/2.4 branch which is still using the WORKSPACE.
I will change to the main branch using MODULE.bazel once torch2.5 nvidia wheel is available for download.